movies-app/src/components/shared/LogOut/LogOut.tsx

```diff
@@ -0,0 +1,14 @@
+"use client";
+import { useRouter } from "next/navigation";
+import { useEffect } from "react";
+
+
+export const LogOut = () => {

```

Comment:
You don't need a dedicated page to log out. Also, no need for router push since AuthRedirect should handle redirecting. Use mutate form swr to update the user cache so that AuthRedirect takes over.